File: submissions/sm_102_amit/week_03/day_3/session_2/count_consonants.js
// count consonants
function countConsonants(input) {
  var count = 0;
  input = input.toLowerCase();
  for (var i = 0; i < input.length; i++) {
    if (
      input[i] == "a" ||
      input[i] == "e" ||
      input[i] == "i" ||
      input[i] == "o" ||
      input[i] == "u" ||
      input[i] == " "
    ) {
      count++;
    }
  }
  var consonants = input.length - count;
  console.log(consonants);
}

countConsonants("happy   birthday");

File: submissions/sm_001_aalind/week_13/day_4/session_1/count_consonants.py
def is_vowel(char):
    if char == "a" or char == "e" or char == 'i' or char == 'o' or char == 'u':
        return True
    else:
        return False

string = input("Enter a string: ")

count = 0

for char in string:
    if !is_vowel(char):
        count += 1

print(count)